config/locales/en.yml

```diff
@@ -868,36 +1595,108 @@ en:
       subscribe: Subscribe
       subscribed_on_another_device: Subscribe on this Device
       unsubscribe: Unsubscribe
+  controllers:
```

Comment:
@prashanth726 Since this new key is being introduces, we'll also need to update the [internationalization guide](https://developers.pupilfirst.com/docs/internationalization/) in developer docs.

Reply:
Done - https://github.com/pupilfirst/pupilfirst/pull/933/commits/0922c691b3601d68b445ac3775ee30bb415a7d1f